articles/clients/index.md

```diff
@@ -4,186 +4,167 @@ description: Explains the basics of creating and using Auth0 Clients.
+- [Settings](${manage_url}/#/clients/${account.clientId}/settings): Lists all the available settings for your client.
 
-Connections may be shared among multiple clients.
+- [Addons](${manage_url}/#/clients/${account.clientId}/addons): Add-ons are extensions associated with clients. They are typically third-party APIs used by the client(s) for which Auth0 generates access tokens. For more details refer to: [Addons](/clients/addons).
 
-## Rules
+- [Connections](${manage_url}/#/clients/${account.clientId}/connections): Connections are sources of users. They are categorized into Database, Social and Enterprise and can be shared among different clients. For more details refer to: [Connections](/clients/connections). For a detailed list on the supported Identity Providers refer to: [Identity Providers Supported by Auth0](/identityproviders).
 
-![](/media/articles/applications/rules-flow.png)
 
-[Rules](/rules) are code snippets written in JavaScript that are executed as part of the Auth0 authentication process. This happens every time a user authenticates. Rules enable very powerful customizations and extensions to be easily added to Auth0.
+Click on the [Settings](${manage_url}/#/clients/${account.clientId}/settings) tab of your client to review the available settings:
 
-Rules enable powerful customizations and allow extensions to be easily added to Auth0.
+- **Name**: The name of your client. This information is editable and you will see in the portal, emails, logs, and so on.
 
-Within the context of a rule, we have access to the Client the user is authenticating to, which is useful if we want to apply coarse-grained authorization policies for our applications:
+- **Domain**: Your Auth0 account name. Note that the domain name is chosen when you create a new Auth0 account and cannot be changed. If you need a different one you have to register for a new account by selecting *New Account* at the top right menu.
 
-* Only HR officials can access Application X
-* Only US-based users can access Application Y
+- **Client ID**: The unique identifier for your client. This is the ID you will use with when configuring authentication with Auth0. It is generated by the system when you create a new client and it cannot be modified.
 
-Here's a [sample rule](https://github.com/auth0/rules/blob/master/rules/simple-user-whitelist-for-app.md) where only the users in the whitelist are allowed to access the application:
+- **Client Secret**: A base64 encoded string used to sign and validate tokens for authentication flows and to gain access to select Auth0 API endpoints. By default, the value is hidden, so check the **Reveal Client Secret** box to see this value.
```

Comment:
This secret we see in the client settings can serve multiple purposes. One one hand, it's going to be use to sign the `id_tokens` (not `access_tokens`) when the algorithm is HS256. On the other hand, it's also "the client password" and be used on the token endpoint for executing several operations (e.g. client grant exchange, code exchange, etc).

Again, this is different from the API secret. The API secret is used for signing `access_tokens` (if using HS256).

The funny thing about client secrets is that are some client types that have no use for it (i.e. SPA / Native). Further down the line we will be removing them from the dashboard for those app types.